model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -157,8 +162,13 @@ private function getWeeksInBetweenDates(DateTime $initDate, DateTime $endDate) {
      * @return float
      * @throws null
      */
-    private function getWorkedHoursInThisJourneyPeriod() {
-        $thisWeekInitDay = DateTime::createFromFormat( 'Y-m-d', date('Y-m-d', strtotime('last monday', $this->currentJourney->getInitDate()->getTimestamp())));
+    private function getWorkedHoursInThisPeriod() {
+        $initWeek = $this->currentUserGoal ?
+            max(
+                $this->currentJourney->getInitDate(), $this->currentUserGoal->getInitDate()
+            ) : $this->currentJourney->getInitDate();
+
+        $thisWeekInitDay = DateTime::createFromFormat( 'Y-m-d', date('Y-m-d', strtotime('last monday', $initWeek->getTimestamp())));
```

Comment:
Imagine the goal starts on Monday, January the 4th: `last monday` will be the Monday from the previous week (Dec, 28) so you will be adding some worked hours that are outside the period.